lib/glimesh_web/templates/about/faq.html.md

```diff
@@ -2,161 +2,151 @@
 
 ## General Questions
 
-##### What's Glimesh?
+##### What’s Glimesh?
 
-Glimesh is a next generation streaming platform built by the community, for the community. We know establishing a following is hard and the hours are long. Our platform focuses on increasing discoverability for content creators and implementing the latest in streaming technology to level the playing field. We understand the importance of interaction between content creators and their fans and we're dedicated to innovating new ways to bring communities closer together.
+Glimesh is a next-generation streaming platform built by the community, for the community. We know establishing a following is hard and the hours are long. Our platform focuses on increasing discoverability for content creators and implementing the latest in streaming technology to level the playing field. We understand the importance of interaction between content creators and their fans and we’re dedicated to innovating new ways to bring communities closer together.
 
 ##### Why make this project?
 
-The core inspiration of this project has been equality. There are two main problems with existing streaming websites: their customers are their shareholders and the gap between streamers is great. Glimesh is our chance to make this right, and build a platform that puts the streamers & community first, not the advertisers.
+The core inspiration of this project has been equality. There are two main problems with existing streaming websites: they take too much of a cut and the gap between streamers is great. Glimesh is our chance to make this right, and build a platform that puts the streamers and community first, not the advertisers.
```

Comment:
I think we changed this line originally because we take the same cut even though we say they take too much of a cut. Can we find something else to change it to?